using-mobile-extensions/adobe-target/migration.md

```diff
@@ -0,0 +1,312 @@
+```
+
+{% endtab %}
+{% endtabs %}
+
+### extensionVersion
+
+{% tabs %}
+{% tab title="AEP 3.x (Swift)" %}
+
+```swift
+static var extensionVersion: String
+```
+
+{% endtab %}
+
+{% tab title="AEP 3.x (Objective-C)" %}
+
+```objc
++ (nonnull NSString*) extensionVersion;
+```
+
+{% endtab %}
+
+{% tab title="ACP 2.x (Objective-C)" %}
+
+```objc
++ (nonnull NSString*) extensionVersion;
+```
+
+{% endtab %}
+{% endtabs %}
+
+### getThirdPartyId
+
+{% tabs %}
+{% tab title="AEP 3.x (Swift)" %}
+
+```swift
+static func getThirdPartyId(_ completion: @escaping (String?, Error?) -> Void)
+```
+
+{% endtab %}
+
+{% tab title="AEP 3.x (Objective-C)" %}
+
+```objc
++ (void) getThirdPartyId:^(NSString* _Nullable, NSError* _Nullable) completion;
+```
+
+{% endtab %}
+
+{% tab title="ACP 2.x (Objective-C)" %}
+
+```objc
++ (void) getThirdPartyId: (nonnull void (^) (NSString* __nullable thirdPartyId)) callback;
+```
+
+{% endtab %}
+{% endtabs %}
+
+### getTntId
+
+{% tabs %}
+{% tab title="AEP 3.x (Swift)" %}
+
+```swift
+static func getTntId(_ completion: @escaping (String?, Error?) -> Void)
+```
+
+{% endtab %}
+
+{% tab title="AEP 3.x (Objective-C)" %}
+
+```objc
++ (void) getTntId:^(NSString* _Nullable, NSError* _Nullable) completion;
+```
+
+{% endtab %}
+
+{% tab title="ACP 2.x (Objective-C)" %}
+
+```objc
++ (void) getTntId: (nonnull void (^) (NSString* __nullable tntId)) callback;
+```
+
+{% endtab %}
+{% endtabs %}
+
+### prefetchContent
+
+{% tabs %}
+{% tab title="AEP 3.x (Swift)" %}
+
+```swift
+static func prefetchContent(_ prefetchArray: [TargetPrefetch], with targetParameters: TargetParameters? = nil, _ completion: ((Error?) -> Void)?)
+```
+
+{% endtab %}
+
+{% tab title="AEP 3.x (Objective-C)" %}
+
+```objc
++ (void) prefetchContent: (NSArray<AEPTargetPrefetchObject*>* _NonNull)
+          withParameters: (AEPTargetParameters* _Nullable)
+                callback: ^(NSError* _Nullable) completion;
+```
+
+{% endtab %}
+
+{% tab title="ACP 2.x (Objective-C)" %}
+
+```objc
++ (void) prefetchContent: (nonnull NSArray<ACPTargetPrefetchObject*>*) prefetchObjectArray
+          withParameters: (nullable ACPTargetParameters*) parameters
+                callback: (nullable void (^) (NSError* _Nullable error)) callback;
+```
+
+{% endtab %}
+{% endtabs %}
+
+### registerExtension
+
+{% tabs %}
+{% tab title="AEP 3.x (Swift)" %}
+{% hint style="info" %}
+Registration occurs by passing `Target` to the `MobileCore.registerExtensions` API.
+{% endhint %}
+
+```swift
+MobileCore.registerExtensions([Target.self])
+```
+
+{% endtab %}
+
+{% tab title="AEP 3.x (Objective-C)" %}
+{% hint style="info" %}
+Registration occurs by passing `AEPMobileTarget` to the `[AEPMobileCore registerExtensions:completion:]` API.
+{% endhint %}
+
+```objc
+[AEPMobileCore registerExtensions:@[AEPMobileTarget.class] completion:nil];
+```
+
+{% endtab %}
+
+{% tab title="ACP 2.x (Objective-C)" %}
+
+```objc
++ (void) registerExtension;
+```
```

Comment:
Here in first two tabs for 3.x you are providing the example usage and in the third tab for 2.x syntax, can we fix this please?